conf/spark-defaults.conf

```diff
@@ -0,0 +1,18 @@
+spark.hadoop.aws.region eu-west-1
+
+spark.hadoop.fs.s3.impl  org.apache.hadoop.fs.s3a.S3AFileSystem
+spark.hadoop.fs.s3a.impl org.apache.hadoop.fs.s3a.S3AFileSystem
+spark.hadoop.fs.s3n.impl org.apache.hadoop.fs.s3native.NativeS3FileSystem
+
+spark.hadoop.fs.s3a.experimental.fadvise random
+
+spark.hadoop.fs.gs.impl com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem
+spark.hadoop.fs.AbstractFileSystem.gs.impl com.google.cloud.hadoop.fs.gcs.GoogleHadoopFS
+
+spark.hadoop.mapreduce.fileoutputcommitter.algorithm.version        2
+spark.hadoop.mapreduce.fileoutputcommitter.cleanup-failures.ignored true
+
+spark.hadoop.parquet.enable.summary-metadata false
+spark.sql.parquet.mergeSchema                false
+spark.sql.parquet.filterPushdown             true
+spark.sql.hive.metastorePartitionPruning     true
```

Comment:
maybe some optimization https://spark.apache.org/docs/latest/cloud-integration.html#committing-work-into-cloud-storage-safely-and-fast and  https://spark.apache.org/docs/latest/cloud-integration.html#parquet-io-settings ?